navigation/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -422,6 +423,8 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
    */
   private void initNavigation() {
     navigation = new MapboxNavigation(this, Mapbox.getAccessToken());
+    MapboxTelemetry.getInstance().newUserAgent(NavigationConstants.NAVIGATION_VIEW_USER_AGENT
+      + BuildConfig.VERSION_NAME);
```

Comment:
@cammace would we need to set the user agent in MapboxNavigation? 

Should we move this into MapboxNavigation with a setter for something like `setUsesDefaultUserInterface()` so that we don't need to import telemetry into the UI part of the library?